src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -175,6 +175,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    type=int, validator=validate_fileupload_sas_ttl,
                    help='The amount of time a SAS URI generated by IoT Hub is valid before it expires,'
                         ' between 1 and 24 hours.')
+        c.argument('min_tls_version', options_list=['--min-tls-version', '--mintls'],
+                   type=str, help='Specifies the minimum TLS version to support for this hub. Can be set to'
```

Comment:
Please write help in the first person voice
```suggestion
                   type=str, help='Specify the minimum TLS version to support for this hub. Can be set to'
```